index.js

```diff
@@ -423,34 +423,18 @@ var rawDocs = {
   communicationProtocols: {
     type: 'tutorials',
     clean: 'Communication Protocols',
-    url: 'https://raw.githubusercontent.com/tessel/docs/master/tutorials/communication-protocols.md',
+    url: 'https://raw.githubusercontent.com/tessel/t2-docs/master/tutorials/communication-protocols.md',
     text: '',
     updated: null,
     newlink: '/docs/communicationProtocols'
   },
   DIYModule: {
     type: 'tutorials',
     clean: 'Making a DIY Module',
-    url: 'https://raw.githubusercontent.com/tessel/docs/master/tutorials/diy_module_creation.md',
+    url: 'https://raw.githubusercontent.com/tessel/t2-docs/master/tutorials/diy_module.md',
```

Comment:
Should be https://raw.githubusercontent.com/tessel/t2-docs/master/tutorials/diy-module.md